tests/acceptance/features/coreApiSharePublicLink1/changingPublicLinkShare.feature

```diff
@@ -10,261 +10,110 @@ Feature: changing a public link share
     And user "Alice" has uploaded file "filesForUpload/textfile.txt" to "PARENT/parent.txt"
 
 
-  Scenario Outline: Public can or can-not delete file through publicly shared link depending on having delete permissions using the public WebDAV API
+  Scenario: Public can or can-not delete file through publicly shared link depending on having delete permissions using the public WebDAV API
     Given user "Alice" has created a public link share with settings
-      | path        | /PARENT       |
-      | permissions | <permissions> |
-    When the public deletes file "parent.txt" from the last public link share using the <public-webdav-api-version> public WebDAV API
-    Then the HTTP status code should be "<http-status-code>"
-    And as "Alice" file "PARENT/parent.txt" <should-or-not> exist
-
-    @notToImplementOnOCIS @issue-ocis-2079
-    Examples:
-      | permissions               | http-status-code | should-or-not | public-webdav-api-version |
-      | read,update,create        | 403              | should        | old                       |
-      | read,update,create,delete | 204              | should not    | old                       |
-
-    @issue-ocis-reva-292
-    Examples:
-      | permissions               | http-status-code | should-or-not | public-webdav-api-version |
-      | read,update,create        | 403              | should        | new                       |
-      | read,update,create,delete | 204              | should not    | new                       |
-
-
-  Scenario Outline: Public link share permissions work correctly for renaming and share permissions read,update,create with the public WebDAV API
-    Given user "Alice" has created a public link share with settings
-      | path        | /PARENT            |
-      | permissions | read,update,create |
-    When the public renames file "parent.txt" to "newparent.txt" from the last public link share using the <public-webdav-api-version> public WebDAV API
-    Then the HTTP status code should be "403"
-    And as "Alice" file "/PARENT/parent.txt" should exist
-    And as "Alice" file "/PARENT/newparent.txt" should not exist
-
-    @notToImplementOnOCIS @issue-ocis-2079
-    Examples:
-      | public-webdav-api-version |
-      | old                       |
+      | path        | /PARENT                   |
+      | permissions | read,update,create,delete |
```

Comment:
Can we also provide permissions other than this? Example for Viewer role

Reply:
yes we can

Reply:
yes, we can check with valid permission 1,5,4